### DISCUSSION TITLE
dangerousRemoteDomainIpcAccess  不适用于IP地址，临时解决方案

### DISCUSSION THREAD

**Discussion by iFTY-R** [tw93/Pake#573] [Q&A] [answered]
https://github.com/tauri-apps/tauri/issues/7009

**Comment by iFTY-R** [comment, marked as answer]
### 找到一个临时解决方案：

修改依赖包的源代码
找到 
C:\Users\%USER_HOME%\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tauri-1.4.1\src\scope\ipc.rs

修改  函数  remote_access_for

 将 155 行中的 matches_domain 删除掉，去掉校验
```
if matches_window && matches_domain && scope.is_none() {
        scope.replace(s.clone());
}
```
改成 
```
if matches_window && scope.is_none() {
        scope.replace(s.clone());
}
```
可以临时用来解决这个问题

**最后记得删除 项目中的 src-tauri/target 文件夹，重新调试或者打包即可**

**Reply by tw93** [reply]
牛逼
